### PR TITLE
Fixed 6

### PR DESCRIPTION
**Pull request by borcuttjahns**
Fixed 6, everything should be working. Let me know if you're happy with this, and if so I can go ahead and start finalizing our responses to the reviewers, namely by describing the new scaling and making concessions where the model isn't doing quite as well. It looks like what parameters we show in figures 1 and 2 might need some work too - if you let me know which parameters you plan on permanently freezing I can quickly parse those down.

### CODE REVIEW

File: ckine/figures/figure6.py [hunk 1]
```diff
@@ -278,6 +281,7 @@ def Spec_Aff(ax, npoints, unkVecAff):
     ax.set_xlabel("Relative IL-2Rβ/γc $K_a$")
     ax.set_ylabel("Specificity")
     ax.set_xlim((10e-3, 10e0))
+    ax.set_yscale("log")
```
Comment (aarmey) [inline ckine/figures/figure6.py:284]: Does this need to be log-scaled?

**Review by aarmey — APPROVED** [review]
One question. Either way this is good to merge. I'll make a comment about something I think we still need to address on the other PR.

File: ckine/figures/figure6.py (hunk 1 above)
Comment (borcuttjahns) [reply ckine/figures/figure6.py:285]: Yea unfortunately it does - now that the NK cell predictions happen to be quite a bit larger than that of Tregs and Thelpers, without the log scale the trend is very difficult to see as NK/Treg is very small.

**Comment by borcuttjahns** [comment]
I fixed the scaling inconsistency between 6 and S7 - turned out to be a dataframe ordering issue. Planning on merging this once the checks pass.

**Pull request merged**